requirements.txt

```diff
@@ -1,10 +1,10 @@
 pyzmq>=17.1.2
-typeguard
+typeguard>=2.5
 ipyparallel
 globus-sdk
 dill
 tblib
 ipykernel
 requests
 paramiko
-psutil>=5.5.1
\ No newline at end of file
+psutil>=5.5.1
```

Comment:
Not sure why it shows there is a change on psutil (but cannot tell any difference).

Reply:
it's an addition/removal of a newline at the end of file from my editor

Reply:
which i've just fixed